### DISCUSSION TITLE
Flutterfire configure is not allowing me to add package name.

### DISCUSSION THREAD

**Discussion by Bobbyjoness** [firebase/flutterfire#12441] [Q&A]
I have an existing flutter and firebase android app that I am trying to repair and migrate. I decide to create a clean flutter project to make things easier. I used flutterfire configure to set up the firebase portion. The problem is that I originally had the wrong applicationID in the build.gradle. the configure process ended up making a new app on the existing firebase project. I fixed the build.gradle and ran flutterfire configure to see if it would pick up the change. Instead it is asking me for the package name but I can't type anything in. I press enter and the configuration process fails because firebase attempts to create a new app without package name. Is there a way to fix this? I am on windows. I typically use command prompt. I could try powershell if it makes any difference.

**Comment by abdullahq19** [comment]
I was adding the project package name from VS Code terminal and had the same issue, what worked for me was just adding the app through command prompt, it let me write the whole package name

**Comment by AndreCosta101** [comment]
I'm having the same issue. Can't add the package name and "flutterfire configure" does not work without it. I'm stuck.

**Reply by abdullahq19** [reply]
You are facing this issue because you are running "flutterfire_configure" inside VS Code terminal, for some reason vs code doesn't allow to write package name in its terminal. Just go to your command prompt and navigate to your project's directory and run the command there. It will let you enter the full package name. 

If it still gives error after entering the full package name, then you probably need to delete the existing project from the firebase console with the incorrect package name and run the command again. Hope it helps!

**Reply by AndreCosta101** [reply]
Thank you!

**Reply by jasminaaa20** [reply]
I still couldn't resolve the issue. Turns out, I hadn't specified the package name in AndroidManifest.xml file. Once I added the package name there, the command identified the package name automatically and now it works.
